src/Sulu/Bundle/DocumentManagerBundle/Bridge/DocumentInspector.php

```diff
@@ -112,7 +113,23 @@ public function getStructureMetadata(StructureBehavior $document)
     {
         return $this->structureFactory->getStructureMetadata(
             $this->getMetadata($document)->getAlias(),
-            $document->getStructureType()
+            $structureType = $document->getStructureType()
+        );
+    }
+
+    /**
+     * Return the default structure for the given StructureBehavior implementing document.
+     *
+     * @param StructureBehavior $document
+     *
+     * @return StructureMetadata
+     */
+    public function getDefaultStructureType(StructureBehavior $document)
```

Comment:
This method doesn't return anything really related to the document, so I would prefer to not have it in the `DocumentInspector`. Also, from what I've seen this method is only used in one place, why not just putting this code there?